### ISSUE TITLE
`valid-jsdoc`'s `prefer` option doesn't work for "return"

### ISSUE THREAD

**Issue by lencioni** [eslint/eslint#2047]
With the following configuration:

```yaml
valid-jsdoc:
  - 2
  -
    prefer:
      return: return
    requireReturn: false
    requireParam: false
    requireReturnDescription: false
```

and the following file:

```js
/**
 * @return {number} test
 */
var test = function() {
  return 1;
};
```

I get the following output:

```
tmp.js
  1:0  error  Use @return instead             valid-jsdoc
  4:4  error  test is defined but never used  no-unused-vars

✖ 2 problems (2 errors, 0 warnings)
```

If I change my tag from `@return` to `@returns`, the error goes away. It seems that this option currently only affects the error message and not the actual check.

ESLint 0.17.0.

**Comment by lencioni** [comment]
I think I misunderstood the documentation. I should be using `returns: return` instead.
